MobiusCore/Source/BackwardsCompatibility.swift

```diff
@@ -90,12 +90,12 @@ public extension MobiusController {
         )
     }
 
-    @available(*, deprecated, message: "use model instead")
+    @available(*, deprecated, renamed: "model")
```

Comment:
In this case, the fixit will change `loop.getModel()` to `loop.model()`, which is wrong, but the resulting error is reasonably clear.